src/asm_ostream.cpp

```diff
@@ -199,12 +198,18 @@ struct InstructionPrinterVisitor {
     void operator()(Exit const& b) { os_ << "exit"; }
 
     void operator()(Jmp const& b) {
+        assert(false);
```

Comment:
Could an eBPF program could be constructed to hit this case or not?
If one could, then I think this should throw a C++ exception instead of just asserting.

Reply:
There indeed shouldn't be an assert here. Not because such a program could be constructed. This method is not currently reachable, but it would be useful (for debugging, probably). I changed it to print absolute jumps.